django_github/management/commands/github_organisation_resync.py

```diff
@@ -0,0 +1,106 @@
+from django.core.management.base import BaseCommand
+from django_github import models as gh_models
+from django_github.utils import _get_token
+import requests
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Ingest Github organisation information, users and teams'
+
+    headers = None
+
+    def handle(self, *args, **options):
+        integrations = gh_models.GithubIntegration.objects.all()
+
+        for integration in integrations:
+            self._set_default_state(integration)
+
+            if not integration.enabled:
+                continue
+
+            GITHUB_TOKEN = _get_token(integration.app_id, integration.secrets, integration.app_installation_id)
```

Comment:
Not a constant anyway :P If it was like a single-time fetched result, it would be ok - but this is not as much.

```suggestion
            github_token = _get_token(integration.app_id, integration.secrets, integration.app_installation_id)
```